sesame/src/main/scala/org/w3/banana/sesame/SesameTransactor.scala

```diff
@@ -0,0 +1,28 @@
+package org.w3.banana.sesame
+
+import org.openrdf.repository.RepositoryConnection
+import org.w3.banana.{Transactor, SparqlUpdate, RDFStore}
+
+import scala.util.{Failure, Try}
+
+trait SesameTransactor extends Transactor[Sesame,RepositoryConnection] {
+
+  def r[T](conn: RepositoryConnection, body: => T): Try[T] = Try{
+    val result = body
+    conn.close()
```

Comment:
and further down...

"Connections should not be left open unless you are using them."

Closing of connections should not be done by a transaction. Perhaps we need the concept of a Session, but that's not for here.